dp/cloud/docker/go/active_mode_controller/Dockerfile

```
@@ -1,4 +1,15 @@
-FROM golang:1.16.14-alpine3.14 AS build
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.18.4-alpine3.16 AS build
```

Easy way for the golang CI check to work for this file:
```suggestion
ARG GOLANG_VERSION="1.18.3"
FROM golang:{GOLANG_VERSION}-alpine3.16 AS build
```

---

@nstng @sebathomas updated AMC Dockerfile, so that is checked by `.github/workflows/scripts/golang_check_version.sh`